AniruddhaKanhere/FreeRTOS-Plus-TCP
# Unit-tests for FreeRTOS_Sockets.c file (#407)

* Initial refactoring commit

* Uncrustified

* Update after some bug fixes

* Fix build issues

* Update build details

* Update with more refactor and first test

* Make builds pass

* Uncrustify/Add unit-test/Remove static qualifier

* Update with ~100% coverage

* ICMP coverage 100%

* Update with some reorganization

* 100% coverage utils and timers

* Add Socket unit-test 70% coverage

* Uncrustify and Add more UT coverage

* 100% coverage

* Uncrustify

* Uncrustify all files

* Try to merge main to this

* Add ignored unit-tests and reorder inclusion order

* Remove commented code from UT

* Remove kernel and stubs from coverage info

* Add briefs to all the functions.

* Update the last functions with briefs

* Uncrustify: triggered by comment.

* Fix spellings

* Break the test file in multiple files

* Fix coverage

* Uncrustify and spell check fix



Description
-----------


Test Steps
-----------


Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
